Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs

```diff
@@ -37,9 +35,17 @@ private void OnTestCompleted(object sender, EventArgs eventArgs)
         {
             _dispatcher.Invoke(() =>
             {
-                if (FindResource("ResultsByOutcome") is CollectionViewSource resource)
+                if (FindResource("ResultsByOutcome") is CollectionViewSource outcome)
                 {
-                    resource.View.Refresh();
+                    outcome.View.Refresh();
+                }
+                if (FindResource("ResultsByModule") is CollectionViewSource module)
+                {
+                    module.View.Refresh();
+                }
+                if (FindResource("ResultsByCategory") is CollectionViewSource category)
+                {
+                    category.View.Refresh();
```

Comment:
Since these are all collectionViews we should be updating the backing collections to fire a PropertyChanged event that will automatically update the UI... I'll try to make sure that happens in my changes to UnitTesting